Frameworks/Foundation/NSURLCredential.mm

```diff
@@ -58,6 +58,10 @@ + (NSURLCredential*)credentialForTrust:(SecTrustRef)trust {
 - (instancetype)initWithIdentity:(SecIdentityRef)identity
                     certificates:(NSArray*)certArray
                      persistence:(NSURLCredentialPersistence)persistence {
```

Comment:
We don't have any of this implemented so I don't know how we could test it beyond passing in nil.  Could be made a todo.

---
In reply to: [73749039](https://github.com/Microsoft/WinObjC/pull/826#discussion_r73749039) [](ancestors = 73749039)